INSTALL.md

```diff
@@ -1209,21 +1200,21 @@ directory and invoking the configuration commands from there.
 
     $ mkdir /var/tmp/openssl-build
     $ cd /var/tmp/openssl-build
-    $ /PATH/TO/OPENSSL/SOURCE/Configure [[ options ]]
+    $ /PATH/TO/OPENSSL/SOURCE/Configure [options...]
 
 #### OpenVMS example
 
     $ set default sys$login:
     $ create/dir [.tmp.openssl-build]
     $ set default [.tmp.openssl-build]
-    $ perl D:[PATH.TO.OPENSSL.SOURCE]Configure [[ options ]]
+    $ perl D:[PATH.TO.OPENSSL.SOURCE]Configure [options...]
```

Comment:
So just FYI, `[options...]` is correct syntax on VMS if you want to look at a directory tree, i.e. the directory `[options]` and all its subdirectories...

Reply:
I worked on VMS decades ago.  Who had a "whoami.dcl" command and by default we included RCS information at the end, so you can imagine what happened with the "$Log$" entries :)